.ci/azure/linux_conditional_compilation.yml

```diff
@@ -88,6 +88,7 @@ jobs:
       cmakeArgs: >
         -GNinja
         -DVERBOSE_BUILD=ON
+        -DCMAKE_COMPILE_WARNING_AS_ERROR=ON
```

Comment:
looks like it has effect only since [cmake 3.24](https://cmake.org/cmake/help/latest/variable/CMAKE_COMPILE_WARNING_AS_ERROR.html) and we still have warnings on Windows with current master